src/OrchardCore.Modules/OrchardCore.Lucene/Views/Search-Results.cshtml

```diff
@@ -0,0 +1,23 @@
+@model dynamic
+@using OrchardCore.ContentManagement;
+@inject IContentItemDisplayManager DisplayManager
+
+@{
+    var contentItems = (IEnumerable<ContentItem>)Model.ContentItems;
```

Comment:
@sebastienros Here is why I asked you on Skype is we could pass a custom ViewModel to a 
`@await DisplayAsync(Model.SearchResults)`
The ViewModel passed here is a Dynamic Shape so I need to cast the items which I don't like much.
If there is a solution for this let me know.